### DISCUSSION TITLE
Remember polls

### DISCUSSION THREAD

**Discussion by danielwagn3r** [lukevella/rallly#941] [Ideas]
As logged in user I would like to remember / watch a poll to have it in my personal dashboard without the need to remember the invite link.

**Comment by danielwagn3r** [comment]
Corresponds to #928 although it’s a different approach

**Comment by NikoHadouken** [comment]
duplicate of https://github.com/lukevella/rallly/discussions/889

**Reply by danielwagn3r** [reply]
> duplicate of #889

Yes and no. I clearly had the polls on which a user voted in mind, but remember a poll without prior voting, like marking is as to be remembered (others would say favorite, star, ...) would be a usecase.

**Comment by danielwagn3r** [comment]
With the changes in Rallly 4 this feature would make even more sense than before.
